packages/search/tests/integration/components/navi-search-result/asset-test.js

```diff
@@ -39,12 +94,50 @@ module('Integration | Component | navi-search-result-asset', function(hooks) {
 
     await render(hbs`<NaviSearchResult::Asset @data={{this.data}} @closeResults={{fn this.closeResults}} />`);
 
-    assert.dom('.navi-search-asset-result').exists({ count: 2 }, '2 results are displayed');
-    let results = findAll('.navi-search-asset-result__item');
-    let expectedResults = ['Revenue report 1', 'Revenue Dashboard'];
+    assert.dom('.navi-search-asset-result').exists({ count: 5 }, '5 results are displayed');
+    assert.equal(
+      find('.navi-search-result-options__button').textContent.trim(),
+      'Show more',
+      'Show more button is shown.'
+    );
+    assert.deepEqual(
+      findAll('.navi-search-asset-result__item').map(result => result.textContent.trim()),
```

Comment:
Not necessary, but we usually keep it short
```suggestion
      findAll('.navi-search-asset-result__item').map(el => el.textContent.trim()),
```